ads/rubicon.js

```diff
@@ -27,90 +24,16 @@ import {loadScript, validateData, writeScript} from '../3p/3p';
 export function rubicon(global, data) {
   // TODO: check mandatory fields
   validateData(data, [], [
-    'slot', 'targeting', 'categoryExclusions',
-    'tagForChildDirectedTreatment', 'cookieOptions',
-    'overrideWidth', 'overrideHeight', 'loadingStrategy',
-    'consentNotificationId', 'useSameDomainRenderingUntilDeprecated',
     'account', 'site', 'zone', 'size',
-    'pos', 'kw', 'visitor', 'inventory',
+    'kw', 'visitor', 'inventory',
     'type', 'method', 'callback',
```

Comment:
Does `type` need to be validated?  I don't see it used anywhere.

Reply:
I think "type" here refers to the vendor the pub is using:
<amp-ad width=320 height=50
    type="rubicon"
    data-method="smartTag"
    data-account="14062"
    data-site="70608"
    data-zone="335918"
    data-size="43">
</amp-ad>

But good point, we don't need to validate that in our js file. Removing.